xmtp_id/src/scw_verifier/mod.rs

```diff
@@ -128,6 +129,15 @@ impl MultiSmartContractSignatureVerifier {
                 info!("No upgraded chain url for chain {id}, using default.");
             };
         });
+
+        if let Ok(url) = env::var("FOUNDRY_URL") {
```

Comment:
Since this is specific to tests + the validation service, WDYT about putting this in `mls_validation_service` under `cfg(test)`? It could go right where we're building the verifier afaiu


PR looks good otherwise!

Reply:
The problem is it's a dev feature, not a test feature. We could put this under a `#[cfg(debug_assertions)]` flag.